apps/dev-recruiters/src/components/modules/FormPage/signUpForm.tsx

```diff
@@ -58,13 +58,20 @@ export default function SignUpForm({
         (number) => /^\d+(\.\d{1,2})?$/.test(number.toString())
       ),
     experience: Yup.string().required('Experience Field Entry is Required'),
-    accepted: Yup.boolean().required('Acceptance Field Entry is Required'),
+    isAgeOver18: Yup.boolean()
```

Comment:
This would require https://github.com/dev-launchers/strapiv4/pull/216 right?

Reply:
Yes. The  PR  will add those new columns.